react-client/src/components/auth/signUp.component.js

```diff
@@ -8,10 +8,11 @@ class SignUp extends Component {
         let form = new FormData(event.target);
         
         axios.post(url, form)
-        .then(function(response){
+        .then(response => {
             console.log(response);
+            this.props.history.push('/login');
         })
-        .catch(function(error){
+        .catch(error => {
             console.log(error)
```

Comment:
Make sure server errors are handled and displayed to user as feedback where necessary